### PR TITLE
Feature/39703 - Aria-labels

### PR DESCRIPTION
**Pull request by Aekal**
Internal: 39703
#310 

* Update `aria-label` values
* Remove unnecessary `aria-label` from dropdown and `this` words in loop

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update label hidden style
 - Refactored `file-upload` component - removed dependecy on button styles, JS code made more roboust
 - Update accessibility features in `file-upload` component
+- Update aria-label values
```
Comment (szafran89) [inline CHANGELOG.md]: Without update will be `enough`

File: components/03-modules/filter/filter--swatch.hbs [hunk 2]
```diff
@@ -14,11 +14,11 @@
         <div class="filter__swatch {{ class }}" {{{ attributes }}}>
             {{#each options }}
                 <a href="{{href}}"
-                   class="filter__swatch-option {{ this.class }}"
-                   aria-label="{{this.ariaLabel}}"
-                   {{{ this.attributes }}}
+                   class="filter__swatch-option {{ class }}"
+                   aria-label="{{ariaLabel}}"
```
Comment (szafran89) [inline components/03-modules/filter/filter--swatch.hbs]: spaces before and after `{{ ariaLabel }}`

File: CHANGELOG.md (hunk 1 above)
Comment (Aekal) [reply CHANGELOG.md]: Ok, done

File: components/03-modules/filter/filter--swatch.hbs (hunk 2 above)
Comment (Aekal) [reply components/03-modules/filter/filter--swatch.hbs]: Added

**Pull request merged**